src/menus/cheat_menu.cpp

```diff
@@ -41,7 +43,8 @@ Line lines[LINES] = {
     {"infinite slingshot", InfiniteSlingshot, "Gives Link 99 slingshot pellets", true, &CheatItems[InfiniteSlingshot].active},
     {"invincible", Invincible, "Makes Link invincible", true, &CheatItems[Invincible].active},
     {"invincible enemies", InvincibleEnemies, "Makes some enemies invincible", true, &CheatItems[InvincibleEnemies].active},
-    {"moon jump", MoonJump, "Hold R+A to moon jump", true, &CheatItems[MoonJump].active}};
+    {"moon jump", MoonJump, "Hold R+A to moon jump", true, &CheatItems[MoonJump].active},
```

Comment:
Would probably change this to "Hold R+A to Moon Jump" given that it is a specific term rather than normal English. This is consistent with how techniques are written in ZSR (though I would still recommend the change either way).